recipes/r-multimode/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = '1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-multimode
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/multimode_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/multimode/multimode_{{ version }}.tar.gz
+  sha256: 9dea73b28162377dc7b4e3a0b99f0b36f850c5e48472963e562a791397145390
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-diptest
+    - r-ks
+    - r-rootsolve
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-diptest
+    - r-ks
+    - r-rootsolve
+
+test:
+  commands:
+    - $R -e "library('multimode')"           # [not win]
+    - "\"%R%\" -e \"library('multimode')\""  # [win]
+
+about:
+  home: https://arxiv.org/abs/1803.00472/
+  license: GPL-3
+  summary: Different examples and methods for testing (including different proposals described
+    in Ameijeiras-Alonso et al., 2018 <DOI:10.1007/s11749-018-0611-5>) and exploring
+    (including the mode tree, mode forest and SiZer) the number of modes using nonparametric
+    techniques.
+  license_family: GPL3
+  license_file: LICENSE
```

Comment:
Please remove the included license file and change this to:
```license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'```

Reply:
@dbast Thanks for the feedback! The skeleton package generation tool originally had it set to that, however, in at least a couple places in the conda-forge docs (https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml and https://github.com/conda-forge/staged-recipes/blob/a504af81c05491bf7b0b018b2fa1efe64767985c/recipes/example/meta.yaml#L52-L55), and in the PR submission template, it was suggested that a physical file be included. Is this no longer the case?

Reply:
This is still the case and `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'` is a reference to a physical file in the r-base package that is then included during build into the created r-multimode package tarball.